### PR TITLE
Correct Scalable Sizing Crash for Various Zone Equipment

### PR DESCRIPTION
**Pull request by RKStrand**
## Pull request overview

This pull request corrects a crash that happened with the scalable sizing routines.  At issue was a variable was not being reset to false consistently across all of the different equipment that use scalable sizing.  This was causing a crash when scalable sizing was being used for a high temperature radiant heater and then a DX coil was being sized.  The unexpected wrong value for a critical flag caused the wrong code to be executed and as a result a hard crash.  This was corrected for the high temperature radiant heater as well as other equipment which use this same flag.  The new code corrects the problem and the user defect input file now runs as it should.  This problem was eliminated for other equipment combinations as well.  The original problem was logged as GitHub Issue #5728.
### Work Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [ x ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
- [ x ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
  - Defect: This pull request repairs a github defect issue #5728.
### Review Checklist

This will not be exhaustively relevant to every PR.
- [x] Code style (parentheses padding, variable names)
- [x] Functional code review (it has to work!)
- [x] If defect, results of running current develop vs this branch should exhibit the fix
- [x] CI status: all green or justified
- [x] Performance: CI Linux results include performance check -- verify this
- [x] Unit Test(s)
- C++ checks:
  - [x] Argument types
  - [ ] If any virtual classes, ensure virtual destructor included, other things
- IDD changes n/a

### CODE REVIEW

**Comment by mjwitte** [comment]
@rraustad Mr. Sizing Guru, do these changes look ok to you?

File: tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc [hunk 1]
```diff
@@ -137,5 +141,35 @@ namespace EnergyPlus {
 
 	}
 
+	TEST_F( EnergyPlusFixture, HighTempRadiantSystemTest_SizeHighTempRadiantSystemScalableFlagSetTest )
+	{
+		int RadSysNum;
+		int SizingTypesNum;
+		
+		HighTempRadiantSystem::clear_state();
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc]: This doesn't need to be here.  I'll remove it.  It gets called from the fixture.

**Review by mjwitte — COMMENTED** [review]
@RKStrand Based on a search of the code for `DataScalableCapSizingOn`, it looks like HVACVariableRefrigerantFlow.cc, PackagedTerminalHeatPump.cc, and WindowAC.cc are missing trailing `DataScalableCapSizingON = false` statements.

File: tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc (hunk 1 above)
Comment (RKStrand) [reply tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc]: @mjwitte I'll look at these and add as appropriate (DataScalableCapSizingON = false).

Comment (RKStrand) [reply tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc]: @mjwitte And thanks for removing the extraneous clear_state.  Still not 100% on my understanding of those, as you can tell.

**Comment by RKStrand** [comment]
@mjwitte I added the proper assignment of the flag to the other models that use the DataScalableCapSizingON flag.  So, I think that addresses your comments, Mike.

**Review by mjwitte — APPROVED** [review]

**Comment by rraustad** [comment]
Just got back from vacation. These changes look correct to me.

**Pull request merged**